CHANGELOG.md

```diff
@@ -2,6 +2,13 @@
 
 All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
 
+## 1.31.0 (2020-03-24)
+
+
+### Bug Fixes
+
+* **cognito:** user pool - link style email verification fails to deploy ([#6938](https://github.com/aws/aws-cdk/issues/6938)) ([b5c60d5](https://github.com/aws/aws-cdk/commit/b5c60d50a6c4fb7e93185c5874a2651ba40d0247)), closes [#6811](https://github.com/aws/aws-cdk/issues/6811)
+
```

Comment:
This fails to mention a lot of things.... Including

> fix(cdk-assets): context path not honored by Docker asset build (#6957)

This is kinda puzzling.

Reply:
Ohhh this could be my bad, in fact... I'm going to try and re-generate this.